docs/optimizing.md

```diff
@@ -1,6 +1,6 @@
 # Optimizing Playbooks
 
-By using all of the following optimizations together its estimated that playbook execution time can be reduced by up to 60% compared to default behavior.
+It is predicted that the combined use of these optimizations can reduce playbook execution time when compared to default behavior. This reduction can be significant in some circumstances.
```

Comment:
Can the word "predicted" be changed to something more concrete?

Reply:
How about one of the following?
1. `Based on analysis the combined use...`
2. `It is predicted based on analysis that the combined use...`
3. `The combined use...`
Any other suggestions?

Reply:
I would go with 3->
The combined use of...

Reply:
Done